src/Web/Sfa.Das.Sas.Web.UnitTests/Application/Handlers/GetStandardHandlerTest.cs

```diff
@@ -66,6 +69,20 @@ public void ShouldReturnFoundStandardInResponse()
             response.Standard.Should().Be(standard);
         }
 
+
+        [Test]
+        public void ShouldReturnFoundStandardAssessmentOrgsInResponse()
+        {
+            var query = new GetStandardQuery() { Id = "1", Keywords = "Test" };
+            var standard = new Standard { StandardId = query.Id };
+            var orgs = new List<Organisation> { new Organisation() };
+            _mockGetStandards.Setup(x => x.GetStandardById(query.Id)).Returns(standard);
+            _mockAssessmentOrgsClient.Setup(x => x.ByStandard(query.Id)).Returns(orgs);
+            var response = _sut.Handle(query);
+
+            response.AssessmentOrganisations.Count.Should().Be(1);
+        }
+
         [Test]
```

Comment:
please add tests to cover other scenarios 
1) Api returns null
`_mockAssessmentOrgsClient.Setup(x => x.ByStandard(query.Id)).Returns((IEnumerable<Organisation>)null);`
2) Api throws exception
`_mockAssessmentOrgsClient.Setup(x => x.ByStandard(query.Id)).Throws(new HttpRequestException());`